test/unittests/libtesteth/blockchainTest.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file blockchainTest.cpp
+ * Unit tests for blockchain test filling/execution.
+ */
+
+#include <boost/test/unit_test.hpp>
+#include <test/tools/libtesteth/TestOutputHelper.h>
+
+using namespace dev::test;
+
+BOOST_FIXTURE_TEST_SUITE(BlockChainTestSuite, TestOutputHelper)
```

Comment:
maybe add a certain indicator. that this suite is coverage for testeth
like CoverBlockchainTest